keyboards/varanidae/keymaps/default/keymap.c

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2023 QMK
+ * Copyright 2023 Silc Renew / Tokage IT Lab.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_BTN3,    KC_F16, KC_F17,  KC_F18,  KC_F19,    KC_F1,     KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9,   KC_F10, KC_F11, KC_F12,  KC_F13, KC_F14, KC_F15,
+        KC_F20,     KC_F21, KC_F22,  KC_F23,  KC_F24,    KC_ESC,    KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,    KC_9,   KC_0,   KC_MINS, KC_EQL, KC_DEL, KC_BSPC,
+        KC_PSLS,    KC_P7,  KC_P8,   KC_P9,   KC_HOME,   KC_TAB,      KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,    KC_O,   KC_P,   KC_LBRC,  KC_RBRC,  KC_ENT,
+        KC_PAST,    KC_P4,  KC_P5,   KC_P6,   KC_END,    KC_LALT,        KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,    KC_L,   KC_SCLN, KC_QUOT,  KC_BSLS,
+        KC_PMNS,    KC_P1,  KC_P2,   KC_P3,   KC_PGUP,   KC_LSFT, KC_BSLS,  KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT, KC_SLSH, KC_GRV,   KC_RSFT,
```

Comment:
The proper mapping for the extra ISO key near the left Shift is `KC_NUBS`.

(Technically the key near the ISO Enter should also be `KC_NUHS` instead of `KC_BSLS` for an ISO keyboard, but most OSes treat those keycodes as mostly equivalent, so using the ANSI mapping for that key is acceptable; the key near the left Shift really must be NUBS though.)
```suggestion
        KC_PMNS,    KC_P1,  KC_P2,   KC_P3,   KC_PGUP,   KC_LSFT, KC_NUBS,  KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT, KC_SLSH, KC_GRV,   KC_RSFT,
```